src/holodeck/environments.py

```diff
@@ -658,7 +669,7 @@ def __windows_start_process__(self, binary_path, task_key, verbose):
                                                        'Global\\HOLODECK_LOADING_SEM' + self._uuid)
         self._world_process = \
             subprocess.Popen([binary_path, task_key, '-HolodeckOn', '-LOG=HolodeckLog.txt',
-                              '-ResX=' + str(self._window_size[1]), '-ResY=' +
+                              '-ForceRes', '-ResX=' + str(self._window_size[1]), '-ResY=' +
```

Comment:
Before if you set an engine larger than the window size it would shrink it down. This forces the window to be whatever size you specify